relayer/src/chain/cosmos/types/tx.rs

```diff
@@ -0,0 +1,18 @@
+use ibc::events::IbcEvent;
+use ibc_proto::cosmos::tx::v1beta1::{AuthInfo, TxBody};
+use tendermint_rpc::endpoint::broadcast::tx_sync::Response;
+
+pub struct SignedTx {
```

Comment:
This is a common struct for both `Tx` and `TxRaw` so that there is no need to have different versions of functions returning slightly different results.